README.md

```diff
@@ -9,3 +9,86 @@ $ bower install gaia-components/gaia-list
 ## Examples
 
 - [Example](http://gaia-components.github.io/gaia-list/)
+
+## Usage
+
+Settings style list
+
+```
+<gaia-list>
+  <a>
+  <i data-icon="brightness"></i>
+  <h3 flex>Menu 1</h3>
+  <i data-icon="forward-light"></i>
+  </a>
+</gaia-list>
+```
+
+List with switch and divider
+
+```
+<style>
+  gaia-list .divided {
+    border-left: solid 1px;
+    padding-left: 14px;
+
+    border-color:
+      var(--border-color,
+      var(--background-plus));
+  }
+ </style>
+
+
+<gaia-list>
+  <li class="ripple">
+    <label flex flexbox for="wifi-switch">
```

Comment:
They do different things. `flex` expands element to fill available space `flex: 1`, `flexbox` defines the element as a `display: flex`.